manga-loader.user.js

```diff
@@ -1810,6 +1859,7 @@ var getViewer = function(prevChapter, nextChapter) {
       var settings = '<table><tr><td>';
       // Custom CSS
       var cssProfiles = storeGet('ml-setting-css-profiles');
+      preloadProfiles(cssProfiles);
```

Comment:
This call needs to be after the following if that creates the default profile or it will crash on a fresh install (since `cssProfiles` will be undefined).